src/lib/eliom_client.client.mli

```diff
@@ -315,6 +327,59 @@ val init : unit -> unit
 
 val set_reload_function : (unit -> unit -> unit Lwt.t) -> unit
 
+(** Store the document/body of the current page, which will be used when going 
+    back to this page in history. 
```

Comment:
Elaborate on "will be used". It should be clear that this causes the page NOT to be charged again.